tests/test_tasks.py

```diff
@@ -201,6 +201,31 @@ def testReduceExposure(self):
         self.assertMaskedImagesEqual(putted["calexp"].maskedImage, self.arc)
         self.assertSpectra(SpectrumSet.fromPfsArm(putted["pfsArm"]), False, checkReferenceLines=False)
 
+    def testExtractSpectraFiberInclusion(self):
+        """Test that the feature of specifying fibers to include
```

Comment:
Sentence doesn't seem complete. Change to 'Tests the feature of specifying fibers to include.'?